src/app/rmm/alias.ts

```diff
@@ -0,0 +1,106 @@
+// --------- BEGIN RUNBOX LICENSE ---------
+// Copyright (C) 2016-2018 Runbox Solutions AS (runbox.com).
+// 
+// This file is part of Runbox 7.
+// 
+// Runbox 7 is free software: You can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the
+// Free Software Foundation, either version 3 of the License, or (at your
+// option) any later version.
+// 
+// Runbox 7 is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with Runbox 7. If not, see <https://www.gnu.org/licenses/>.
+// ---------- END RUNBOX LICENSE ----------
+import { timeout, share } from 'rxjs/operators';
+import { RMM } from '../rmm';
+
+export class Alias {
+    public profiles:any;
+    public aliases: any;
+    public aliases_unique: any;
+    public aliases_counter: any;
+    constructor(
+        private app: RMM,
+    ) {
+    }
+    load() {
```

Comment:
I'm not a fan of having to `load()` things manually after having created an object in order to have the data available. If there's no reason to load them every time, then perhaps it'd be better to load them automatically whenever someone tries to use the data. Something like this:

```
private _profiles: any;
get profiles(): any {
    if (!this.data_loaded) {
        this.load(); // sets this.data_loaded
    }
    return _profiles;
}
```

Right now it's an object that requires extra steps to be used correctly, and otherwise might lead to subtle bugs that won't be easy to track down, because nothing was actually done.

Reply:
when you add this hack, leave load there so we can re-load it